common/OpTestHost.py

```diff
@@ -423,11 +350,11 @@ def host_check_config(self, i_kernel, i_config):
     #
     def host_check_pkg_for_utility(self, i_oslevel, i_cmd):
         if 'Ubuntu' in i_oslevel:
-            l_res = self._ssh_execute("dpkg -S `which %s`" % i_cmd, timeout=60)
+            l_res = self.ssh.run_command("dpkg -S `which %s`" % i_cmd, timeout=60)
             return l_res
         else:
             l_cmd = "rpm -qf `which %s`" % i_cmd
-            l_res = self._ssh_execute(l_cmd, timeout=60)
+            l_res = self.ssh_run_command(l_cmd, timeout=60)
```

Comment:
I mean here there is a typo s/self.ssh_run_command/self.ssh.run_command/
We need dot instead of underscore